docs/api/target/index.rst

```diff
@@ -7,16 +7,20 @@ Target visualizers specialize in visually describing the dependent variable for
 
 The following visualizations are currently implemented:
 
+-  :doc:`binning`: Generate histogram with vertical lines showing the recommended value point to bin data into evenly distributed bins.
 -  :doc:`class_balance`: Visual inspection of the target to show the support of each class to the final estimator.
 -  :doc:`feature_correlation`: Plot correlation between features and dependent variables.
 
 .. code:: python
 
     # Target Visualizers Imports
-    from yellowbrick.classifier import FeatureCorrelation, ClassBalance
+    from yellowbrick.target import BalancedBinningReference
```

Comment:
Deleted the first import line because it had `yellowbrick.classifier` instead of `yellowbrick.target`.

Then I added `BalancedBinningReference` to the list of Target Visualizer Imports